lms/static/sass/utilities/_variables.scss

```diff
@@ -33,7 +33,8 @@ $font-bold: 700;
 
 // MISC: base fonts/colors
 $body-font-size: em(14);
-$body-line-height: golden-ratio(.875em, 1);
```

Comment:
NOTE: ``golden-ratio`` is deprecated in the newer version of Bourbon. Moving to ``modular-scale`` (using golden-ratio setting).